prow/cmd/deck/main.go

```diff
@@ -473,6 +483,13 @@ func handleBadge(ja *jobs.JobAgent) http.HandlerFunc {
 	}
 }
 
+// handleJobHistory handles requests to get the history of a given job
+// The url must look like this:
+//
+// /job-history/<gcs-bucket-name>/pr-logs/directory/<job-name>
```

Comment:
`pr-logs` is not necessarily part of the URL, see https://prow.k8s.io/job-history/kubernetes-jenkins/logs/ci-kubernetes-e2e-prow-canary for example

Reply:
Updated